laterpay/asset_sources/js/laterpay-backend-account.js

```diff
@@ -199,6 +200,14 @@
                     )
                     .done(function() {
                         $o.requestSent = false;
+
+                        if ( 'laterpay_plugin_mode' === form_id ) {
+                            var pluginStatus = $o.pluginModeToggle.prop('checked') ? 'Live' : 'Test';
+                            var commonLabel = lpVars.gaData.sandbox_merchant_id + ' | ' +
+                                lpVars.gaData.site_url + ' | ';
+                            var eveCategory = 'LP WP Account', eveAction = 'Account Status Change';
```

Comment:
should put a new variable in a new line for better readability, in this case `eveAction`.